src/push_service/response.rs

```diff
@@ -68,6 +68,27 @@ where
             })?;
             Err(ServiceError::ProofRequiredError(proof_required))
         },
+        StatusCode::LENGTH_REQUIRED => {
+            #[derive(Debug, serde::Deserialize)]
+            struct LinkedDeviceNumberError {
```

Comment:
the problem of doing this is that if any other endpoint gets 411 we'll deserialize the response to this type, this has happened with `CONFLICT` before. I wonder what we could do that works better? maybe infer the possible custom error type with a template parameter?